scripts/globals/spells/blue/reactor_cool.lua

```diff
@@ -25,10 +25,10 @@ spell_object.onMagicCastingCheck = function(caster, target, spell)
 end
 
 spell_object.onSpellCast = function(caster, target, spell)
-    local typeEffectOne = xi.effect.ICE_SPIKES
-    local typeEffectTwo = xi.effect.DEFENSE_BOOST
-    local powerOne = 5
-    local powerTwo = 12
+    local typeEffectOne = xi.effect.DEFENSE_BOOST
```

Comment:
Perhaps these variables could be renamed to reflect the effect type?

Reply:
There are Five other Blue Magic scripts which follow this convention. Seems pretty straight forward and easy to read as-is, imo. 

Can you give me an example of how you're envisioning the variable names?

Reply:
The same you say? I take it back, that smells like we should be moving spells like this to the global and tabling data

Reply:
Sounds like subsequent PR territory, that concept is outside of the scope of this one.

Reply:
Yup. I'll take a look this weekend for some ideas to clean things up